### PR TITLE
Example of `/etc/sysconfig/flanneld` is not valid

### PR DESCRIPTION
**Pull request by alexanderilyin**
Configuration keys were different, I got those errors when followed steps from docs:
```
[ailyin@elite ~]$ systemctl status flanneld.service
● flanneld.service - Flanneld overlay address etcd agent
   Loaded: loaded (/usr/lib/systemd/system/flanneld.service; enabled; vendor preset: disabled)
   Active: activating (start) since Wed 2017-01-04 19:00:21 EST; 34s ago
 Main PID: 4668 (flanneld)
   Memory: 3.5M
   CGroup: /system.slice/flanneld.service
           └─4668 /usr/bin/flanneld -etcd-endpoints= -etcd-prefix=

Jan 04 19:00:46 elite.dev flanneld[4668]: E0104 19:00:46.395562 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
Jan 04 19:00:47 elite.dev flanneld[4668]: E0104 19:00:47.395788 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
Jan 04 19:00:48 elite.dev flanneld[4668]: E0104 19:00:48.395990 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
Jan 04 19:00:49 elite.dev flanneld[4668]: E0104 19:00:49.396141 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
Jan 04 19:00:50 elite.dev flanneld[4668]: E0104 19:00:50.396306 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
Jan 04 19:00:51 elite.dev flanneld[4668]: E0104 19:00:51.396451 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
Jan 04 19:00:52 elite.dev flanneld[4668]: E0104 19:00:52.396572 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
Jan 04 19:00:53 elite.dev flanneld[4668]: E0104 19:00:53.396701 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
Jan 04 19:00:54 elite.dev flanneld[4668]: E0104 19:00:54.396855 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
Jan 04 19:00:55 elite.dev flanneld[4668]: E0104 19:00:55.396992 04668 network.go:53] Failed to retrieve network config: client: etcd cluster is unavailable or misconfigured
```

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes.github.io/2135)

### CODE REVIEW

**Comment by jaredbhatti** [comment]
Can you sign the CLA, please? :)

File: docs/getting-started-guides/centos/centos_manual_config.md [hunk 1]
```diff
@@ -131,15 +131,17 @@ $ etcdctl mk /kube-centos/network/config "{ \"Network\": \"172.30.0.0/16\", \"Su
...
 ```shell
+# Flanneld configuration options
+
 # etcd url location.  Point this to the server where etcd runs
-FLANNEL_ETCD="http://centos-master:2379"
+FLANNEL_ETCD_ENDPOINTS="http://centos-master:2379"
 
 # etcd config key.  This is the configuration key that flannel queries
 # For address range assignment
-FLANNEL_ETCD_KEY="/kube-centos/network"
```
Comment (thockin) [inline docs/getting-started-guides/centos/centos_manual_config.md]: This same var appears later in the file, too

**Comment by lxpollitt** [comment]
@tomdee can you chip in on this issue (when you are feeling better!)

File: docs/getting-started-guides/centos/centos_manual_config.md (hunk 1 above)
Comment (alexanderilyin) [inline docs/getting-started-guides/centos/centos_manual_config.md]: @thockin fixed in c4fa6d9. It was same example but for `node`. In my case I had both `master` and `node` on one host so I've skipped that step.

**Comment by alexanderilyin** [comment]
@jaredbhatti, I signed it!

**Comment by alexanderilyin** [comment]
Some version details, not sure if you need it.

```
/usr/bin/flanneld -version
0.5.5
```
```
cat /etc/redhat-release
CentOS Linux release 7.3.1611 (Core)
```
```
etcd --version
etcd Version: 2.3.7
Git SHA: fd17c91
Go Version: go1.6.3
Go OS/Arch: linux/amd64
```

```
kubectl version
Client Version: version.Info{Major:"1", Minor:"3", GitVersion:"v1.3.0", GitCommit:"86dc49aa137175378ac7fba7751c3d3e7f18e5fc", GitTreeState:"clean", BuildDate:"2016-12-15T16:57:18Z", GoVersion:"go1.6.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"3", GitVersion:"v1.3.0", GitCommit:"86dc49aa137175378ac7fba7751c3d3e7f18e5fc", GitTreeState:"clean", BuildDate:"2016-12-15T16:57:18Z", GoVersion:"go1.6.3", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by tomdee** [comment]
@thockin The vars are dynamically created - see https://github.com/coreos/flannel#environment-variables and https://github.com/coreos/flannel#key-command-line-options.

So this change looks good to me as a flannel maintainer.

**Pull request merged**